src/ServiceControl.Monitoring/Infrastructure/EndpointMessageType.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace ServiceControl.Monitoring.Infrastructure
+{
+    public class EndpointMessageType
+    {
+        public EndpointMessageType(string endpointName, string messageType)
```

Comment:
This might get more complicated in the real world. If I have two messages of the same name but different assembly versions, should they be combined or shown separately? 

I think this is good enough for now though and it's really nicely abstracted so we can mess with it later.

Reply:
IMO this should be done either in SC.Monitoring or maybe client itself (it's an easy aggregation).

Reply:
As a side note: currently the string holds the type full name so we do have assembly, version and public key and can parse that in SC.Monitoring.